v3/src/components/case-table/case-table.scss

```diff
@@ -266,6 +266,11 @@ $table-body-font-size: 8pt;
           background-color: #D3E9C8;
         }
       }
+      &[aria-selected=true] {
+        .codap-data-cell {
+          background-color: var(--rdg-row-selected-background-color) !important;
```

Comment:
We should try to avoid use of `!important`. I ran it locally to see why it's necessary and I think it can just be removed in this case.